app/javascript/dashboard/routes/dashboard/settings/inbox/components/CampaignTable.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <section class="campaigns-table-wrap">
+    <ve-table
+      :fixed-header="true"
+      max-height="calc(100vh - 11.4rem)"
+      scroll-width="150rem"
+      :columns="columns"
+      :table-data="tableData"
+      :border-around="true"
+    />
+
+    <empty-state v-if="showEmptyResult" :title="$t('CAMPAIGN.LIST.404')" />
+    <div v-if="isLoading" class="campaign--loader">
+      <spinner />
+      <span>{{ $t('CAMPAIGN.LIST.LOADING_MESSAGE') }}</span>
+    </div>
+  </section>
+</template>
+
+<script>
+import { mixin as clickaway } from 'vue-clickaway';
+import { VeTable } from 'vue-easytable';
+
+import Spinner from 'shared/components/Spinner.vue';
+import Thumbnail from 'dashboard/components/widgets/Thumbnail.vue';
+import EmptyState from 'dashboard/components/widgets/EmptyState.vue';
+import WootSubmitButton from 'dashboard/components/buttons/FormSubmitButton';
```

Comment:
We are not using this  think.

Reply:
`WootSubmitButton` used outside of vue template

Reply:
I think `<woot-button>` might as we have used it in contact table or import `WootButton` here. As we are going to remove `WootSubmitButton`.